libs/feature/search/src/lib/utils/service/fields.service.ts

```diff
@@ -70,6 +72,8 @@ export class FieldsService {
     q: new FullTextSearchField(),
     license: new LicenseSearchField(this.injector),
     owner: new OwnerSearchField(this.injector),
+    qualityScore: new SimpleSearchField('qualityScore', this.injector, 'desc'),
+    territories: new TranslatedSearchField(undefined, this.injector, 'asc'),
```

Comment:
Hum, not sure if there is a cleaner solution for this (cc @jahow), but I'd rename `territories` to `customTranslatedSearchField` here as this is not limited to the territories use case.